x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_suggestions.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SuggestionRequest, VisualizationSuggestion, TableSuggestion } from '../types';
+import { State } from './types';
+
+/**
+ * Generate suggestions for the metric chart.
+ *
+ * @param opts
+ */
+export function getSuggestions(
+  opts: SuggestionRequest<State>
+): Array<VisualizationSuggestion<State>> {
+  return opts.tables
+    .filter(
+      ({ isMultiRow, columns }) =>
+        // We only render metric charts for single-row queries. We require a single, numeric column.
+        !isMultiRow && columns.length === 1 && columns[0].operation.dataType === 'number'
+    )
+    .map(table => getSuggestion(table));
+}
+
+function getSuggestion(table: TableSuggestion): VisualizationSuggestion<State> {
+  const col = table.columns[0];
+  const title = col.operation.label;
+
+  return {
+    title,
+    score: 1,
+    datasourceSuggestionId: table.datasourceSuggestionId,
+    previewIcon: 'visMetric',
```

Comment:
Totally optional, but maybe it makes sense to have a preview expression here? Not too useless IMHO, especially if we add a short (3 words max) title directly into the suggestion tile:
<img width="239" alt="Screenshot 2019-08-12 at 13 04 51" src="https://user-images.githubusercontent.com/1508364/62860859-dc4a8900-bd01-11e9-9df6-53e40fad04cb.png">

(hacked together, but this is how it could look like if we bring _short_ suggestion titles back)
<img width="183" alt="Screenshot 2019-08-12 at 13 07 52" src="https://user-images.githubusercontent.com/1508364/62860972-36e3e500-bd02-11e9-9f63-fb014a11bc68.png">

Reply:
Yes. Totally agree.

Reply:
Incidentally, this might be a good reason to keep the autoscale logic in.